Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:stretch
+
+ENV LANG=C.UTF-8
+
+RUN set -x \
+  && apt-get update \
+  && apt-get install -y \
+    build-essential \
+    libacl1 \
+    libacl1-dev \
+    libssl-dev \
+    liblz4-dev \
+    liblz4-1 \
```

Comment:
just noticed that our docs need improving here. also check if you are missing something (that might be present, but better do it explicitly):

https://borgbackup.readthedocs.io/en/stable/installation.html#debian-ubuntu

we shall get libzstd(-dev) installed from dist also (prefer that over our builtin zstd code).